monkey/monkey_island/cc/services/attack/technique_reports/__init__.py

```diff
@@ -111,10 +112,21 @@ def get_tech_base_data(cls):
         data.update({'status': status,
                      'title': title,
                      'message': cls.get_message_by_status(status)})
+        data.update(cls.get_mitigation_by_status(status))
         return data
 
     @classmethod
     def get_base_data_by_status(cls, status):
         data = cls.get_message_and_status(status)
         data.update({'title': cls.technique_title()})
+        data.update(cls.get_mitigation_by_status(status))
         return data
+
+    @classmethod
+    def get_mitigation_by_status(cls, status: ScanStatus) -> dict:
+        if status == ScanStatus.USED.value:
```

Comment:
Perhaps we want `SCANNED` as well? Better to err on the side of safety.

Reply:
If it's scanned/unused there's nothing to mitigate/attack technique already mitigated. Or did I misinterpreted your comment?

Reply:
If it's scanned that doesn't mean that you 100% mitigated just that you protected what Monkey tested for - perhaps you'd like to do more.